core-models/src/main/java/com/anytypeio/anytype/core_models/ProcessEvent.kt

```diff
@@ -0,0 +1,16 @@
+package com.anytypeio.anytype.core_models
+
+sealed class ProcessEvent {
```

Comment:
we could put into Progress data class in the following moment (this way two classes are linked):

```
data class Process {

     sealed class Event {...}
}
```

Reply:
done